apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -112,21 +112,21 @@ public sealed class ErrorReportingExceptionLogger
         }
 
         /// <summary>
-        /// Asynchronously reports an exception that occurred to the Stackdriver Error Reporting API.
+        /// Asynchronously logs an exception that occurred to the Stackdriver Error Reporting API.
         /// </summary>
         /// <returns>A task containing an empty response on success.</returns>
-        public async Task<ReportErrorEventResponse> ReportAsync(HttpContext context, Exception exception)
+        public async Task<ReportErrorEventResponse> LogAsync(HttpContext context, Exception exception)
```

Comment:
Since you're changing "Report" to "Log", it seems that `ReportErrorEventResponse` should be changed to `LogErrorEventResponse`.